src/components/forms/newListingForm.jsx

```diff
@@ -14,12 +15,17 @@ class NewListingForm extends Component{
         this.state = this.initialState;
     }
    
-    initialState = {
+    initialState = {/*
         title: '',
         price: '',
         city: '',
         category: '',
-        description: '',
+        description: '',*/
+        name: '',
+        p_name: '',
+        price: '',
+        distance: '',
+        description: ''
```

Comment:
Very important to update all the key fields. The backend was developed completely separately so in order for the frontend and backend to work seamlessly and obtain all proper information the keys in the database object and frontend have to match. LGTM